src/components/LandingProjectAbout/index.tsx

```diff
@@ -47,50 +33,26 @@ export const LandingProjectAbout: FC = () => (
         <div className='md:col-span-3 xl:col-span-1'>
           <div className='text-white max-w-none leading-7'>
             <p className='max-w-none'>
-              Die Plattform <em>Stadtpuls</em> ist eine prototypische Offene
-              Datenplattform, welche{" "}
-              <strong className='text-green'>
-                Sensordaten von &amp; für Berlin
-              </strong>{" "}
-              als Open Data sammelt, frei zugänglich macht und der Stadt Berlin
-              auf ihrem Weg zur smarten Stadt der Zukunft helfen soll.
+              Mit <em>Stadtpuls</em> haben wir eine offene Plattform geschaffen,
+              die dabei unterstützt, Sensordaten zu erfassen, zu visualisieren
+              und für weitere Datenauswertung bereitzustellen. Wie laut ist es
+              in meiner Straße? Wie hoch ist der CO2-Wert in meinem Klassenraum?
+              Wie hoch ist die Feinstaubbelastung vor meiner Haustür?
+              <br />
+              <em>Stadtpuls</em> hilft, deine Stadt zu erklären.
             </p>
             <p className='mt-4'>
-              Ziel der Plattform ist es Sensordaten der Stadt Berlin
-              aufzubereiten, zu visualisieren und für alle Akteure der
-              Stadtgesellschaft frei und kostenlos zur Verfügung zu stellen.
-              Nutzer:innen können sich über einzelne IoT-Projekte informieren,
-              selber Projekte und Sensoren anlegen oder Daten aus Projekten für
-              die eigene Analyse nutzen. Dabei spielen Klimadaten, aber auch
-              städtische Sensordaten ein große Rolle. Diese Daten in einer
-              gemeinsamen Datenbasis vorzuhalten, zugänglich zu machen und zu
-              visualisieren ist ein erster Schritt in Richtung einer smarten,
-              vernetzten Stadt.
+              So wie es Github für Open Source Software gibt, stellt{" "}
```

Comment:
Not sure if we can compare with GitHub and if we should leave out Bitbucket, gitlab, sourcehut as if GitHub where the only ones.

Reply:
Ja das stimmt, hier geht's aber eher um die Catchphrase. Wenn's okay ist, merge ich erstmal und mache mir aber noch Gedanken, wie man das neutraler ausdrücken könnte. Danke Dir.